virtcontainers/qemu_arm64.go

```diff
@@ -45,7 +45,7 @@ var kernelParams = []Param{
 
 var kernelRootParams = []Param{
 	{"root", "/dev/pmem0p1"},
-	{"rootflags", "data=ordered,errors=remount-ro rw"},
+	{"rootflags", "data=ordered,errors=remount-ro ro"},
```

Comment:
Might be worth adding a comment here explaining why arm64 doesn't use one of the common variables?

Reply:
ok. I will add comments asap. ;)